.github/workflows/release.yml

```diff
@@ -37,6 +47,7 @@ jobs:
               run: yarn version:publish
               env:
                   NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+                  SOLDEER_LOGIN_FILE: ${{ github.workspace }}/soldeer_login
```

Comment:
use of this env var undocumented in the soldeer readme.
I found it in the source code directly
https://github.com/mario-eth/soldeer/blob/c3d70821ed7ef7a3d9264a6872139446c48cac5a/crates/core/src/utils.rs#L62